### PR TITLE
Manoverboard fixes 

### LINKED ISSUE #146: Manoverboard NPE
**Describe the bug**
Manoverboard fails with an NPE from the getTimeFromOverboard when there is no data in the overboards set.

### LINKED ISSUE #147: Manoverboard to released craft
**Describe the bug**
Manoverboarding to a released craft.

**To Reproduce**
1. Pilot craft
2. Release craft
3. Manoverboard to released craft (not sure, this was fairly buggy)

### LINKED ISSUE #148: Manoverboard memory leak
**Describe the bug**
The weak hash map overboards can keep a reference to a sunken or released craft around, hogging memory.

### LINKED ISSUE #149: Remove overboarding to another person's craft
Overboarding to another player's craft has been fairly buggy throughout movecraft's history and caused many bugs.  It appears best to remove the feature and simplify the codebase.

### LINKED ISSUE #150: Manoverboarding to craft
**Describe the bug**
Manoverboarding doesn't work correctly when you haven't jumped off of a craft.

### LINKED ISSUE #153: Hitbox utils
**Describe the bug**
Some hitbox utils are named HitBox and some are named Hitbox, which should become standard? @cccm5

### PR DESCRIPTION
**Pull request by TylerS1066**
### Related issues:
- fix #146
- fix #147 
- fix #148 
- close #149
- fix #150
- fix #153 

### Checklist
- [x] Unit tests 
- [x] Proper internationalization
- [x] Compiled/tested

### CODE REVIEW

**Comment by TylerS1066** [comment]
Update: #150 is fixed with a bypass, once #154 is fixed it will need to be changed.

**Comment by TylerS1066** [comment]
#151 will be a different PR about crew signs, and therefore I consider this PR ready for review!

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/ManOverboardCommand.java [hunk 1]
```diff
@@ -43,17 +40,19 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
         }
 
         if ((System.currentTimeMillis() - CraftManager.getInstance().getTimeFromOverboard(player)) / 1_000 > Settings.ManOverboardTimeout) {
-            player.sendMessage(MOVECRAFT_COMMAND_PREFIX + I18nSupport.getInternationalisedString("ManOverboard - Timed Out"));
-            return true;
-
+            if(!MathUtils.locationNearHitBox(craft.getHitBox(), player.getLocation(), 1))
+            {
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/ManOverboardCommand.java]: Don't use C style brackets. Also, 2 nested if statements should be merged

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/AntiAircraftDirectorSign.java [hunk 2]
```diff
@@ -34,7 +34,7 @@ public final void onSignClick(PlayerInteractEvent event) {
         Craft foundCraft = null;
         CraftManager.getInstance().getCraftsInWorld(block.getWorld());
         for (Craft tcraft : CraftManager.getInstance().getCraftsInWorld(block.getWorld())) {
-            if (MathUtils.locationInHitbox(tcraft.getHitBox(), event.getClickedBlock().getLocation()) &&
+            if (MathUtils.locationInHitBox(tcraft.getHitBox(), event.getClickedBlock().getLocation()) &&
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/AntiAircraftDirectorSign.java]: Whenever you use `locationInHitBox`, you probably actually want to use `locIsNearCraftFast`. This adds a buffer of 3 blocks around the craft such that players onto or next to the craft are considered "in" the craft

**Pull request merged**